src/riak_pb_messages.csv

```diff
@@ -50,6 +50,10 @@
 81,DtFetchResp,riak_dt
 82,DtUpdateReq,riak_dt
 83,DtUpdateResp,riak_dt
+90,TsQueryReq,riak_kv
```

Comment:
Not terribly important, but for the record you managed not to collide with the message identifiers for our pending changes to create a coverage API